feature-libs/quote/components/quote-actions-by-role/quote-actions-by-role.component.spec.ts

```diff
@@ -154,13 +154,32 @@ describe('QuoteActionsByRoleComponent', () => {
     expect(facade).toBeDefined();
   });
 
+  xdescribe('Ghost animation', () => {
+    it('should render view for ghost animation', () => {
+      mockQuoteDetails$.next(null);
+      fixture.detectChanges();
+
+      CommonQuoteTestUtilsService.expectElementPresent(
+        expect,
+        htmlElem,
+        '.cx-ghost-actions'
+      );
+
+      CommonQuoteTestUtilsService.expectNumberOfElementsPresent(
+        expect,
+        htmlElem,
+        '.cx-ghost-button',
+        3
+      );
+    });
+  });
+
   it('should read quote details state', (done) => {
     component.quoteDetails$.pipe(take(1)).subscribe((state) => {
       expect(state).toEqual(mockQuote);
       done();
     });
   });
-
```

Comment:
i think now you not just removed the empty lines you added, but also additional ones. Can you please restore the formatting, so there are no empty line changes?  May just copy the file from the epic and re-add (copy from your change the test)? thx.